projects/addon-doc/components/demo/demo.style.less

```diff
@@ -177,23 +180,41 @@
     }
 }
 
-.t-group,
 .t-checkbox {
     margin-left: 0.75rem;
 }
 
 .t-button,
+.t-copy-code,
 .t-form-controls {
     margin-top: 0.75rem;
 }
 
+.t-code-container {
+    background: var(--tui-base-01);
+    border-radius: 0 0 var(--tui-radius-m) var(--tui-radius-m);
+}
+
+.t-code {
```

Comment:
Doesn't seem like any of these styles do anything. If they are not needed — don't forget to drop the class from the template.

Reply:
This is not need anymore after redesign. Fixed.